### ISSUE TITLE
[0.5.x] stream and subscription not responding

### ISSUE THREAD

**Issue by yschimke** [ReactiveSocket/reactivesocket-java#211]
Run against TcpPongServer

```
package io.reactivesocket.transport.tcp;

import io.reactivesocket.Payload;
import io.reactivesocket.ReactiveSocket;
import io.reactivesocket.client.ReactiveSocketClient;
import io.reactivesocket.frame.ByteBufferUtil;
import io.reactivesocket.transport.tcp.client.TcpTransportClient;
import io.reactivesocket.util.PayloadImpl;
import io.reactivex.Flowable;
import org.reactivestreams.Publisher;
import org.slf4j.event.Level;

import java.net.InetSocketAddress;

import static io.reactivesocket.client.KeepAliveProvider.never;
import static io.reactivesocket.client.SetupProvider.keepAlive;

public class TcpPongClient {
    public static void main(String[] args) {
        Publisher<? extends ReactiveSocket> c = ReactiveSocketClient.create(TcpTransportClient.create(new InetSocketAddress(7878)).logReactiveSocketFrames("rs", Level.WARN),
                keepAlive(never()).disableLease()).connect();

        ReactiveSocket client = Flowable.fromPublisher(c).blockingFirst();

        Publisher<Payload> x = client.requestStream(new PayloadImpl("Hello"));

        Payload response = Flowable.fromPublisher(x).blockingFirst();

        System.out.println(response);
    }
}
```

Output

```
28 Dec 2016 09:40:09,986  INFO [main] (Frame.java:61) - Creating thread pooled named io.reactivesocket.frame.UnpooledFrame
28 Dec 2016 09:40:10,778  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x56236355, L:/192.168.0.21:52919 - R:0.0.0.0/0.0.0.0:7878] Writing frame: Frame[0] => Stream ID: 0 Type: SETUP Version: 0 keep-alive interval: 2147483647 max lifetime: 2147483645 metadata mime type: application/x.reactivesocket.meta+cbor data mime type: application/binary Payload: 
28 Dec 2016 09:40:10,952  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x56236355, L:/192.168.0.21:52919 - R:0.0.0.0/0.0.0.0:7878] Writing frame: Frame[0] => Stream ID: 3 Type: REQUEST_N RequestN: 2147483647 Payload: 
```

**Comment by yschimke** [comment]
Looks like it is caused by https://github.com/ReactiveSocket/reactivesocket-java/pull/210

**Comment by yschimke** [comment]
After:

```
28 Dec 2016 11:11:28,730  INFO [main] (Frame.java:61) - Creating thread pooled named io.reactivesocket.frame.UnpooledFrame
28 Dec 2016 11:11:29,186  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x5f39dedd, L:/192.168.0.21:53919 - R:0.0.0.0/0.0.0.0:7878] Writing frame: Frame[0] => Stream ID: 0 Type: SETUP Version: 0 keep-alive interval: 2147483647 max lifetime: 2147483645 metadata mime type: application/x.reactivesocket.meta+cbor data mime type: application/binary Payload: 
28 Dec 2016 11:11:29,310  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x5f39dedd, L:/192.168.0.21:53919 - R:0.0.0.0/0.0.0.0:7878] Writing frame: Frame[0] => Stream ID: 3 Type: REQUEST_STREAM Initial Request N: 1 Payload: data: "Hello"
28 Dec 2016 11:11:29,312  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x5f39dedd, L:/192.168.0.21:53919 - R:0.0.0.0/0.0.0.0:7878] Writing frame: Frame[0] => Stream ID: 3 Type: NEXT_COMPLETE Payload: 
28 Dec 2016 11:11:29,315  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x5f39dedd, L:/192.168.0.21:53919 - R:0.0.0.0/0.0.0.0:7878] Writing frame: Frame[0] => Stream ID: 3 Type: REQUEST_N RequestN: 2147483647 Payload: 
28 Dec 2016 11:11:29,322  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x5f39dedd, L:/192.168.0.21:53919 - R:0.0.0.0/0.0.0.0:7878] Read frame: Frame[0] => Stream ID: 3 Type: NEXT Payload: data: "��9��:��7��1�ΑG�
g����&,����j�L��}p�ܢ
:�h3Y�..��dt�\���-���A�Ǿ5�kۃ5���l�,׳ީ�7U( 9<=E��	
��q�F���6ubז��9�Qc���� 鏎<�Mr�i���p��(d�{
���?xEB�gv+d$�e6*�"�Opt����F$��?�#F��hC��1����݆�jj���S��<?��uy3$�b��Ew�S��`ۼƵ��fA.�� i�Bo{��(�D:z&B
��j�hS�d��e[7$J�,gH��_-��wD�ؗ�z��k ���bU��[	�J����4W׽1o+k�Q �yDs(`,I��$�«����$�f�:7���41N>
�T�Z|�5�L���~�1B�<6��L���N��h@���Mf&wU5�z��:���R��$fJް7�ꂤ�%�g!쨲`��`qf$'bo(V����/�"��$]���_��[���!C2����$f\�h�	7|�/6U�<��ɪ��*�"��ً�e����/7%Py��ps&"Smt:w*�=�Ζ��m#�G�O���ք�t8~���@[�x&�`23m1
�4�h�z�\�z�ۼr
�Ka�x�@45b����K�ct�?U��.ab�}�/�tw���eCY*A,y���	�"
28 Dec 2016 11:11:29,324  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x5f39dedd, L:/192.168.0.21:53919 - R:0.0.0.0/0.0.0.0:7878] Writing frame: Frame[0] => Stream ID: 3 Type: CANCEL Payload: 
28 Dec 2016 11:11:29,325  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x5f39dedd, L:/192.168.0.21:53919 - R:0.0.0.0/0.0.0.0:7878] Writing frame: Frame[0] => Stream ID: 3 Type: REQUEST_N RequestN: 2147483647 Payload: 
Frame[0] => Stream ID: 3 Type: NEXT_COMPLETE Payload: 
28 Dec 2016 11:11:29,327  WARN [rxnetty-nio-eventloop-1-1] (ReactiveSocketFrameLogger.java:57) - [id: 0x5f39dedd, L:/192.168.0.21:53919 - R:0.0.0.0/0.0.0.0:7878] Read frame: Frame[0] => Stream ID: 3 Type: NEXT_COMPLETE Payload: 
```

**Comment by NiteshKant** [comment]
Fixed via #210
